test/netlib/tservers.py

```diff
@@ -107,3 +108,67 @@ def teardown_class(cls):
     @property
     def last_handler(self):
         return self.server.server.last_handler
+
+
+class SocksHandler(tcp.BaseHandler):
```

Comment:
socksipy is just a socks client, it's not a socks server. May I use mitmproxy.proxy.socks_proxy to test this?